tests/SpecTests/AtlasDataLakeSpecTest.php

```diff
@@ -237,13 +240,8 @@ public function testConnectwithSCRAMSHA256(): void
 
     private function isAtlasDataLake(): bool
     {
-        $cursor = $this->manager->executeCommand(
-            $this->getDatabaseName(),
-            new Command(['buildInfo' => 1])
-        );
-
-        $document = current($cursor->toArray());
+        $isDataLake = getenv('MONGODB_IS_DATA_LAKE');
 
-        return ! empty($document->dataLake);
+        return $isDataLake !== false ? filter_var($isDataLake, FILTER_VALIDATE_BOOLEAN) : false;
```

Comment:
Noted that this code was adapted from `MongoDB\Tests\FunctionalTestCase::isServerless()`.